vtex-apps/login
# add prop showIconProfile and iconLabel

#### What is the purpose of this pull request?
as the title says.

#### What problem is this solving?
Allow the user to change what is written in the login button and hide/show the icon of the login.

[full description](https://app.clubhouse.io/vtex-dev/story/11395/when-changing-the-label-for-login-the-name-of-the-user-doesn-t-show-up).

#### How should this be manually tested?
1. [Access this workspace](https://mariana--storecomponents.myvtex.com/).
2. Access `/admin` 
3. Select `Header` > `Login`
4. Make your changes.
![image](https://user-images.githubusercontent.com/20481790/57932179-b4798f80-7890-11e9-9d05-cb9b7102cd9d.png)
5. The changes should be here
![image](https://user-images.githubusercontent.com/20481790/57932379-2c47ba00-7891-11e9-8cc4-9ce8e5ae53d8.png)

6. If the field `iconLabel` is empty, should be replaced with the default label  `Sign In`

#### Types of changes
